READMEs/datatokens-flow.md

```diff
@@ -77,47 +81,61 @@ python
 In the Python console:
 
 ```python
-import os
+# Create Ocean instance
 from ocean_lib.example_config import ExampleConfig
 from ocean_lib.ocean.ocean import Ocean
-from ocean_lib.web3_internal.wallet import Wallet
-
-private_key = os.getenv('TEST_PRIVATE_KEY1')
 config = ExampleConfig.get_config()
 ocean = Ocean(config)
+
 print(f"config.network_url = '{config.network_url}'")
 print(f"config.block_confirmations = {config.block_confirmations.value}")
+print(f"config.metadata_cache_uri = '{config.metadata_cache_uri}'")
+print(f"config.provider_url = '{config.provider_url}'")
 
-print("Create wallet: begin")
-wallet = Wallet(ocean.web3, private_key, config.block_confirmations, config.transaction_timeout)
-print(f"Create wallet: done. Its address is {wallet.address}")
+# Create Alice's wallet
+import os
+from ocean_lib.web3_internal.wallet import Wallet
+alice_private_key = os.getenv('TEST_PRIVATE_KEY1')
+alice_wallet = Wallet(ocean.web3, alice_private_key, config.block_confirmations, config.transaction_timeout)
+print(f"alice_wallet.address = '{alice_wallet.address}'")
+
+# Mint OCEAN
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+mint_fake_OCEAN(config)
+assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
 
+# Publish an NFT token
 print("Create ERC721 data NFT: begin.")
-erc721_token = ocean.create_nft_token(name="Dataset name", symbol="dtsymbol", from_wallet=wallet)
+erc721_token = ocean.create_nft_token('NFTToken1', 'NFT1', alice_wallet)
```

Comment:
Change `create_nft_token()` --> `create_data_nft`. Perhaps create a new issue for this, as it's changing the interface of ocean.py. 

Change `erc721_token` --> `erc721_nft`

Reply:
@trentmc I have created an issue/PR for this: https://github.com/oceanprotocol/ocean.py/pull/702